### ISSUE TITLE
WCAG 2.2 Violation - The custom combobox element contains invalid, static, non-hidden text elements.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#17]

**Comment by ivrouwe-ubc** [comment]
Closing this Issue, as I've determined that the whole custom combobox/listbox present in the interface will need an overhaul (instead of us remediating discrete issues).
